src/Nox.ClientApi.Tests/Generated/Nox.Generator/Nox.Generator.NoxCodeGenerator/Application.Factories.StoreOwnerFactory.g.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Generated
+﻿using System;// Generated
```

Comment:
generated should be in a single line in a top

Reply:
Using statement was redundant based on the latest implementation and it was removed